File: php/delete_subuser.php
<?php

// Information obtained from authentication

$userId = '';
$token = '';
$subUserId = 0; // Retrieved from Get Subusers

$url = 'https://api.smartproxy.com/v1/users/' . $userId . '/sub-users/' . $subUserId;

$ch = curl_init();

curl_setopt($ch, CURLOPT_URL, $url);
curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
curl_setopt($ch, CURLOPT_CUSTOMREQUEST, "DELETE");

$headers = array();
$headers[] = "Accept: application/json";
$headers[] = "Authorization: Token $token";
curl_setopt($ch, CURLOPT_HTTPHEADER, $headers);

$result = curl_exec($ch);
if (curl_errno($ch)) {
    echo 'Error:' . curl_error($ch);
}
curl_close($ch);
echo $result;

File: csharp/delete_subuser.cs
using System;
using System.IO;
using System.Net;

namespace Smartproxy_API
{
    class delete_subuser
    {
        static void Main(string[] args)
        {
            // Details from authentication
            string userId = "";
            string token = "";
            int sub_user_id = null; // Recieved from get_subusers.cs

            string uri = "https://api.smartproxy.com/v1/users/" + userId + "/sub-users/" + sub_user_id;

            HttpWebRequest request = (HttpWebRequest)WebRequest.Create(uri);
            request.AutomaticDecompression = DecompressionMethods.GZip | DecompressionMethods.Deflate;
            request.PreAuthenticate = true;
            request.Headers.Add("Authorization", "Token " + token);
            request.ContentType = "application/json";
            request.Method = "DELETE";

            using (HttpWebResponse response = (HttpWebResponse)request.GetResponse())
            using (Stream stream = response.GetResponseStream())
            using (StreamReader reader = new StreamReader(stream))
            {
                Console.WriteLine(reader.ReadToEnd());
                Console.ReadKey();
            }
        }
    }
}
